test/validation/api/buffer/buffer.c

```diff
@@ -59,7 +59,7 @@ static void test_pool_alloc_free(const odp_pool_param_t *param)
 	uint32_t i;
 	uint32_t num_buf = 0;
 	void *addr;
-	odp_event_subtype_t subtype;
+	odp_event_subtype_t subtype = ODP_EVENT_NO_SUBTYPE;
```

Comment:
For better coverage subtype could be initialized to an invalid value for the `if (subtype != ODP_EVENT_NO_SUBTYPE)` check below. (e.g. `ODP_EVENT_PACKET_BASIC`).